### PR TITLE
Update use of ErrorListBadRequest

### PR DESCRIPTION
**Pull request by courtneyeh**

### CODE REVIEW

File: data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/PostAttestation.java [hunk 1]
```diff
@@ -87,7 +72,10 @@ public PostAttestation(
                         .getAttestationSchema()
                         .getJsonTypeDefinition()))
             .response(SC_OK, "Attestations are stored in pool and broadcast on appropriate subnet")
-            .response(SC_BAD_REQUEST, "Errors with one or more attestations", BAD_REQUEST_RESPONSE)
+            .response(
+                SC_BAD_REQUEST,
+                "Errors with one or more attestations",
+                ErrorListBadRequest.getFailuresTypeDefinition())
```
Comment (rolfyone) [inline data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/PostAttestation.java]: nit: should probably call this `getJsonTypeDefinition`?

Comment (courtneyeh) [reply data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/PostAttestation.java]: But there are two different type definitions on that class. Can't call them both `getJsonTypeDefinition`

File: data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/schema/ErrorListBadRequest.java [hunk 2]
```diff
@@ -67,4 +72,31 @@ public int hashCode() {
   public String toString() {
     return "ErrorListBadRequest{" + "message='" + message + '\'' + ", errors=" + errors + '}';
   }
+
+  public static SerializableTypeDefinition<ErrorListBadRequest> getStacktraceTypeDefinition() {
+    return SerializableTypeDefinition.object(ErrorListBadRequest.class)
+        .name("StacktraceErrorListBadRequest")
+        .withField("code", INTEGER_TYPE, ErrorListBadRequest::getCode)
+        .withField("message", STRING_TYPE, ErrorListBadRequest::getMessage)
+        .withField(
```
Comment (rolfyone) [inline data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/schema/ErrorListBadRequest.java]: will be `withOptionalField` I think? im not sure we ever send stacktrace...

Comment (courtneyeh) [reply data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/schema/ErrorListBadRequest.java]: In the cases it is used we do

**Review by rolfyone — APPROVED** [review]
LGTM

**Pull request merged**